manifests/y/Yandex/Chats/YandexMessenger/2.170.0/Yandex.Chats.YandexMessenger.locale.ru.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Yandex.Chats.YandexMessenger
+PackageVersion: 2.170.0
+PackageLocale: ru

```

```suggestion
PackageLocale: en-US
```

PackageLocale should follow BCP-47 format. If this package is mainly used in Russia, then change the locale to `ru-RU` and provide ShortDescription in Russian. Otherwise we can make `en-US` as the default locale